canopen/objectdictionary/eds.py

```diff
@@ -356,15 +356,15 @@ def export_common(var, eds, section):
     def export_variable(var, eds):
         if isinstance(var.parent, ObjectDictionary):
             # top level variable
-            section = "%04X" % var.index
+            section = f"{var.index:04X}"
         else:
             # nested variable
-            section = "%04Xsub%X" % (var.index, var.subindex)
+            section = f"{var.index:04X}sub{var.subindex:X}"
 
         export_common(var, eds, section)
-        eds.set(section, "ObjectType", "0x%X" % VAR)
+        eds.set(section, "ObjectType", f"0x{VAR:X}")
```

Comment:
Why is this capitalized? Did it ever work?

Reply:
Probably not.

Reply:
Ha ha, easy to confuse with the local variable `var`... Thank you, now it's clear with looking at the full code :grin: